static/scss/course-cards.scss

```diff
@@ -0,0 +1,105 @@
+// sass-lint:disable mixins-before-declarations
+.course-section h1 {
+  font-size: 44px;
+  font-weight: bold;
+  letter-spacing: 0;
+  line-height: 54px;
+  text-align: center;
+  padding-top: 54px;
+  padding-bottom: 54px;
+  margin: 0 auto;
+  max-width: 760px;
+
+  @include media-breakpoint-down(sm) {
+    font-size: 30px;
+    line-height: 40px;
+    padding-top: 20px;
+    padding-bottom: 22px;
+  }
+}
+
+.course-listing {
+  padding: 0;
+  margin: 0 -22px;
+  list-style: none;
+  letter-spacing: -0.32em;
+
+  .course-card-description {
+    overflow-y: scroll;
```

Comment:
I don't know if we want internal scrollbars here to begin with, but this is making an empty scrollbar show up in Mac Chrome when the contents do not overflow. To get rid of that, this needs be be `overflow-y: auto;`, but I think it would be better just to set a `min-height` for the text section and let it grow vertically if the text is too long.